en/getting_started/README.md

```diff
@@ -54,19 +54,20 @@ When installing, you must minimally select all Visual C++ components as shown:
 You **need to install Qt as described below** instead of using pre-built packages from say, a Linux distribution, because *QGroundControl* needs access to private Qt headers.
 
 To install Qt:
+
 1. Download and run the [Qt Online Installer](http://www.qt.io/download-open-source)
    - **Ubuntu:** 
      - Set the downloaded file to executable using: `chmod +x`. 
      - Install to default location for use with **./qgroundcontrol-start.sh.** If you install Qt to a non-default location you will need to modify **qgroundcontrol-start.sh** in order to run downloaded builds.
-1. In the installer *Select Components* dialog choose: {{ book.qt_version }} (on *Ubuntu* choose Qt 5.11.3).
+1. In the installer *Select Components* dialog choose: {{ book.qt_version }}.
    
    Then install just the following components: 
    - **Windows**: *MSVC 2015 32 bit*
    - **MacOS**: *macOS*
    - **Linux**: *Desktop gcc 64-bit*
    - All:
-     - *Qt Charts* and *Qt Remote Objects (TP)*
-     - *Android ARMv7* (to build Android)
+   - *Qt Charts* and *Qt Remote Objects (TP)*
```

Comment:
This should have remained indented. I'll merge and fix,

Reply:
Oops, left some crap from trying to figure out why the list wasn't rendering properly.

Reply:
OK. So right now this renders as below. If it needs to be anything else, let me know :-)

![image](https://user-images.githubusercontent.com/5368500/62169975-28461700-b36d-11e9-99b6-db9c444d3dd6.png)